README.md

```diff
@@ -67,14 +67,16 @@ There are some formatter config exposed through the Jupyter Lab Advanced Setting
 Add an extra entry to your keyboard shortcuts settings with something like
 
 ```
-{"jupyterlab_code_formatter:black":{
-    "command": "jupyterlab_code_formatter:black",
-    "keys": [
-        "Ctrl K",
-        "Ctrl M"
-    ],
-    "selector": ".jp-Notebook.jp-mod-editMode"
-}}
+{
+    "shortcuts":[{
+        "command": "jupyterlab_code_formatter:autopep8",
```

Comment:
`jupyterlab_code_formatter:autopep8` here would make the README incoherent, because it says `jupyterlab_code_formatter:black` below.

```suggestion
        "command": "jupyterlab_code_formatter:black",
```